ui/components/component-library/form-text-field/README.mdx

```diff
@@ -0,0 +1,278 @@
+import { Story, Canvas, ArgsTable } from '@storybook/addon-docs';
+
+import { FormTextField } from './form-text-field';
+
+# FormTextField
+
+The `FormTextField` allows users to enter text to search. It wraps the `TextField` component that adds a search icon to the left of the input.
+
+<Canvas>
+  <Story id="ui-components-component-library-form-text-field-form-text-field-stories-js--default-story" />
+</Canvas>
+
+## Props
+
+The `FormTextField` accepts all props below as well as all [Box](/docs/ui-components-ui-box-box-stories-js--default-story#props), [TextField](/docs/ui-components-component-library-text-field-text-field-stories-js--default-story#props) and [TextFieldBase](/docs/ui-components-component-library-text-field-base-text-field-base-stories-js--default-story#props) component props
+
+<ArgsTable of={FormTextField} />
```

Comment:
I guess I need to add all the different seeing as I am exposing them and passing them to each child component sigh.

- [ ] Add all missing propTypes

Also weird that these are showing up as `unknown`

<img width="1115" alt="Screen Shot 2022-11-14 at 6 14 19 PM" src="https://user-images.githubusercontent.com/8112138/201794805-0cbd86e5-c8aa-47ba-81d5-aed415618abd.png">

Reply:
I think we have a resolution for this now! Which is to include the children component ArgTables in the parent component MDX documentation!